src/models/launch_lite_1/external-overview.md

```diff
@@ -0,0 +1,12 @@
+# Launch Lite (External Overview)
+
+## Top overview:
+
+![Top Overview](./img/overhead.webp)
+
+## Back overview:
+
+![Back Ports](./img/ports-back.png)
+
+## Box contents overview:
+![Launch Box Contents](./img/launch-lite-box-contents.png)
```

Comment:
Can these accessories be replaced or purchased from System76?

Reply:
I'm pretty sure the tool and cables can be purchased/replaced at the discretion of support, probably the lifter bar too. The extra keys may depend on what we currently have on hand since we purchase the keys as complete sets from our supplier.

Reply:
It's usually be at discretion of production an availability of parts. Individual key caps are not stocked so purchase would be for full set, for example. The lead time on some of the parts for the launch is limiting.

Reply:
Do you think it's worth mentioning that many of these accessories can be sourced by the user if lost or broken (I assume the USB cables are not a unique specification, and that none of these tools are proprietary). I know key pullers are easy to come by, but I'm not sure about the switch puller. Maybe there's a more appropriate place to mention it, but I couldn't find anything on the product page or in the support docs.

Reply:
How about:

> - Contact [System76 Support](https://support.system76.com/) to purchase replacement accessories. Availability may vary.